### ISSUE TITLE
Info about stopped development

### ISSUE THREAD

**Issue by erri120** [erri120/Playnite.Extensions#72]
As you might have seen, the last commit with actual code was b6485db3345e0203075c05463b18fbc7cd36bd9d on 26 Oct 2020. I have stopped development on this project because I'm waiting for Playnite to upgrade to .NET 5 or .NET 6. If you don't know what that means, here is a little explaination:

Playnite is currently using .NET Framework 4.6.2 which is more than 5 years old. In these 5 years a lot of things have changed and .NET Framework is considered legacy and you should not target it anymore. Instead you should target the new .NET 5 or .NET 6 which is currently still in preview. The problem with Playnite specifically is that it is a Desktop Application using WPF (Windows Presentation Foundation), a framework which is, not only Windows only, but also soon to be considered legacy. The successor to WPF is MAUI which comes with .NET 6 but MAUI is designed to be cross-platform (Windows, macOS, iOS, Android) and not cross-operating system (Windows, Linux, macOS). The alternative would be Avalonia which is similar to WPF but made for .NET 5/6 which can target Windows, Linux and macOS, however it is still in early alpha and doesn't have a 1.0 release.

Tl;dr: Playnite can't be upgraded to .NET 5/6 without a major rework of the UI (and other systems). Making it available on Windows, Linux and macOS is even harder so this will likely take some time.

**Comment by Drakses** [comment]
Hi! Any update? They moved or not? With version 9. None of the extensions are working anymore.

**Comment by erri120** [comment]
> Hi! Any update? They moved or not? With version 9. None of the extensions are working anymore.

Playnite still has not made the transition to .NET 6, you can follow this issue if you want to be up-to-date: https://github.com/JosefNemec/Playnite/issues/1141

**Comment by Logerfo** [comment]
WPF is still [supported](https://github.com/dotnet/wpf#status) on .NET 6.
